app/javascript/App/bundles/common/components/Icon.tsx

```diff
@@ -45,19 +45,29 @@ export default ({
   )
 }
 
-const IconWrapper = ({ onClick, children, className }: IconWrapperProps) => {
+const IconWrapper = ({
+  onClick,
+  children,
+  style,
+  className,
+}: IconWrapperProps) => {
   const klass = className || ''
   if (onClick) {
     return (
       <button
         data-heap="news-add-coin-to-watchlist-button"
         className={`btn-icon icon ${klass}`}
         onClick={onClick}
+        style={style}
```

Comment:
Probably for next time, but material ui lets you use the `classes` property on components decorated with `withStyles(...)` to override `className` props. Essentially allowing you to do what you do with `style` but with more control